files/bootstrap.sh

```diff
@@ -360,7 +367,7 @@ if [[ -z "${B64_CLUSTER_CA}" ]] || [[ -z "${APISERVER_ENDPOINT}" ]]; then
             --region=${AWS_DEFAULT_REGION} \
             --name=${CLUSTER_NAME} \
             --output=text \
```

Comment:
@cartermckinnon not for this PR, but I checked and our AMIs come with jq, so I wonder if we could get rid of `output=text` and just rely on jq rather than `awk` since the line split can be so easy to mess up.

Reply:
Agreed!

Reply:
Yeah, this will make the code simple.